src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -108,6 +102,20 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
   }
 
   isPasteAllowed(target: Zone[], clipboardOption?: ClipboardOptions): CommandResult {
+    let updatedCells: ClipboardCell[][] = [];
+    this.cells.forEach((cell) => {
+      let rowCells: ClipboardCell[] = [];
+      cell.forEach((rowCell) => {
+        rowCells = this.getCellData(rowCells, rowCell.position);
+      });
+      updatedCells.push(rowCells);
+    });
```

Comment:
Use `for(const .. of ...)` rather than forEach, and use "const" instead of "let".